_rules/SC1-3-1-aria-required-owned-element.md

```diff
@@ -0,0 +1,243 @@
+---
+name: ARIA required owned elements
+
+description: |
+	The rule checks that a role has at least one of its required owned elements.
```

Comment:
--> "This rule checks that each role has at least one of its required owned elements."